diff --git a/node_modules/@smarttools/eslint-plugin-rxjs/index.cjs b/node_modules/@smarttools/eslint-plugin-rxjs/index.cjs
index e6c238d..074ed0d 100644
--- a/node_modules/@smarttools/eslint-plugin-rxjs/index.cjs
+++ b/node_modules/@smarttools/eslint-plugin-rxjs/index.cjs
@@ -39,7 +39,6 @@ var require_recommended = __commonJS({
     "use strict";
     module2.exports = {
       plugins: ["@smarttools/rxjs"],
-      name: "@smarttools/rxjs/recommended-legacy",
       rules: {
         "@smarttools/rxjs/no-async-subscribe": "error",
         "@smarttools/rxjs/no-create": "error",
@@ -345026,7 +345025,6 @@ var plugin = {
 if (plugin.configs) {
   Object.assign(plugin.configs, {
     recommended: {
-      name: "@smarttools/rxjs/recommended",
       plugins: {
         "@smarttools/rxjs": plugin
       },
